mps.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/la5nta/pat/internal/cmsapi"
+)
+
+const (
+	mpsUsage = `subcommand [options]
+
+subcommands:
+  list [--all]    List message pickup stations for your callsign, or all MPS with --all
+  clear           Delete all message pickup stations for your callsign
+  add [CALLSIGN]  Add a message pickup station`
+
+	mpsExample = `
+  list            List your message pickup stations
+  list --all      List all message pickup stations
+  clear           Delete all your message pickup stations
+  add W1AW        Add W1AW as a message pickup station`
+)
+
+func mpsHandle(ctx context.Context, args []string) {
+	mycall := fOptions.MyCall
+	if mycall == "" {
+		fmt.Println("ERROR: MyCall not configured")
+		os.Exit(1)
+	}
+
+	switch cmd, args := shiftArgs(args); cmd {
+	case "list":
```

Comment:
I think we should keep a local copy of the current list, so the user can review it when they are offline. I imagine there will be a lot of users that don't keep note of their pickup stations.

We could for instance attempt to fetch from the API, write to disk if successful, and fallback to reading from disk. 

What do you think?

Reply:
Thats a good point. It's better to direct users to use INQUIRY when we have that in place 👍